### DISCUSSION TITLE
Windows Start Folder

### DISCUSSION THREAD

**Discussion by skoomabwoy** [jamland/samplescope#157] [General]
Hey,

I'm not sure if it's by design, but on Windows the software is placing its link into "Andy Burkovetskiy" folder in Start menu.
![image](https://user-images.githubusercontent.com/76624205/106353101-6c123780-62e8-11eb-87fb-3ca40df9d245.png)

If you want to promote this name as a brand, that totally makes sense. However, right after installation I had hard time finding the app in the menu, since I was looking it under "Samplescope" name.

As a side note: great app, thank you so much.

**Comment by jamland** [comment]
Hi! 

Thank you for mentioning this.
Yes, this is not by design. 
Obviously, should be placed within `Samplesope` folder.

I will investigate it.
